work/flow.rst

```diff
@@ -217,11 +217,11 @@ Community contributors can of course use personal repositories (and forks of LSS
 When an external user has created a pull request (which should be automatically checked for style using GitHub Actions), a staff member should follow these steps to help the community member merge their contribution:
 
     1. Review the community member's pull request, following our :ref:`code review guidelines<workflow-code-review>`.
-    2. Create a jira ticket (``DM-98765`` in this example) for the contribution, and a local branch of the respective repository.
+    2. Create a jira ticket (``DM-98765`` in this example) for the contribution, and create a local branch of the respective repository (without checking it out yet).
```

Comment:
Actually, in my testing it seems that creating the branch is not necessary; the `fetch` is sufficient.

Reply:
Thanks KT - I've switched around the fetch and the checkout steps to capture this, which now reads better.